atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClient.java

```diff
@@ -75,22 +93,58 @@ public void batch_mutate(Map<ByteBuffer, Map<String, List<Mutation>>> mutation_m
             throws InvalidRequestException, UnavailableException, TimedOutException, TException {
         checkLimitAndCall(() -> {
             super.batch_mutate(mutation_map, consistency_level);
+
+            qosMetrics.updateWriteCount();
+            qosMetrics.updateBytesWritten(getApproximateWriteByteCount(mutation_map));
```

Comment:
we could combine these two steps in a `QosMetrics#recordBytesWritten` method (since they always need to happen together)

Reply:
Done.